File: backend/src/main/java/com/shieldx/securities/dto/SecurityTypeResponse.java
package com.shieldx.securities.dto;

import lombok.Data;

@Data
public class SecurityTypeResponse {
    private int stId;
    private String levelName;
    private String description;
    private String isArmed;
    private double pricePerDay;

    public SecurityTypeResponse(int stId, String levelName, String description, String isArmed, double pricePerDay) {
        this.stId = stId;
        this.levelName = levelName;
        this.description = description;
        this.isArmed = isArmed;
        this.pricePerDay = pricePerDay;
    }
}

File: frontend/shieldx-frontend/src/app/dto/security-type-response.ts
export interface SecurityTypeResponse {
   stId: number;          // Unique identifier for the security type
  levelName: string;    // Matches SecurityTypeRequest
  description: string;  // Matches SecurityTypeRequest
  isArmed: string;      // Matches SecurityTypeRequest (e.g., "yes" or "no")
  pricePerDay: number;  // Matches SecurityTypeRequest
}
